golem-service-base/src/model.rs

```diff
@@ -2620,6 +2623,41 @@ pub struct InterruptResponse {}
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Ord, PartialOrd, Serialize, Deserialize, Object)]
 pub struct ResumeResponse {}
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Ord, PartialOrd, Serialize, Deserialize, Object)]
+pub struct UpdateResponse {}
```

Comment:
as this is in common services namespace i would consider to rename it to WorkerUpdateResponse

Reply:
Hm makes sense, but currently it follows the pattern of the other ones. (See `ResumeResponse` for example above)

Reply:
yes i noticed this part, maybe then we should do also `ResumeResponse` -> `ResumeWorkerResponse` , ....